### ISSUE TITLE
Check object properties

### ISSUE THREAD

**Issue by DrChrispoper** [hpersson/eslint-plugin-jsx-expressions#3]
Could the plugin be updated to catch properties of an object that match the rule?
For example
```
code: [
        'const first = { foo: 0 };',
        "<App>{first.foo && <Foo/>}</App>",
      ]
```
Won't throw an error when 'ideally' it should.

**Comment by hpersson** [comment]
Thank you for bringing this to my attention!
This should be fixed in version 1.3, which has just been released.

**Comment by DrChrispoper** [comment]
Oh wow. Thanks, great timing 😄
